packages/tooltip/src/vaadin-tooltip.js

```diff
@@ -84,6 +88,17 @@ class Tooltip extends ThemePropertyMixin(ElementMixin(PolymerElement)) {
         value: false,
       },
 
+      /**
+       * Position of the tooltip with respect to its target.
+       * Supported values: `top-start`, `top`, `top-end`,
+       * `bottom-start`, `bottom`, `bottom-end`, `start-top`,
+       * `start`, `start-bottom`, `end-top`, `end`, `end-bottom`.
+       */
+      position: {
+        type: String,
+        value: 'bottom',
```

Comment:
IIRC we wanted to use `top` for input fields, should it be the default for all components?
This makes it a bit weird if an element with a tooltip is placed at the top of the page.

Reply:
Can this position handle fallback variants? So that it checks if there is enough space on top and otherwise goes somewhere else?